contact.html

```diff
@@ -4,8 +4,82 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
+    <link rel="stylesheet" href="index.css">
 </head>
 <body>
+    <header>
+        <div>
+            Alex Reber's Site!
+        </div>
+    </header>
+    <nav>
+        <div>
+            <a href="index.html">Home</a> 
+            <a href="contact.html">Contact</a> 
+            <a href="gallery.html">Gallery</a> 
+        </div>
+    </nav>
+    <div class="container">
+        <div style="padding-top: 5%; padding-bottom: 5%">
+          <div style="height: 50px; background-color: #f44336">
+            <p style="color:white">Please fill out your information below</p>
+          </div>
+        <div style="background-color: #ffebee; padding-left: 5%; padding-right: 5%; padding-top: 3%">
+          <div>
+                   <label for="name">Name*</label>
+                   <input name="custName" id="name" type="text" placeholder="First and Last Name" required />
```

Comment:
The 'name' and 'id' attributes serve the same purpose.  Using both results in using up potential 'id' values unnecessarily.  The 'name' attribute is specifically for 'input' tags, so consider avoiding 'id' attribute values on 'input' tags.